src/commands/compile/compile.ts

```diff
@@ -41,6 +47,33 @@ export async function compile(target: Target, forceCompile = false) {
 
   await compileModule.compileJobsServicesTests(target)
 
+  if (target?.macroFolders?.length) {
+    await asyncForEach(
+      target.macroFolders,
+      async (macroFolder: string) => await copyTestMacroFiles(macroFolder)
+    )
+
+    const buildMacroTestFolder = path.join(
+      (await getConstants()).buildDestinationTestFolder,
+      'macros'
+    )
+
+    if (await folderExists(buildMacroTestFolder)) {
+      const macroTestFiles = await (
+        await listFilesAndSubFoldersInFolder(buildMacroTestFolder)
+      ).filter((item: string) => isSasFile(item))
```

Comment:
Just a little note - when you're applying a filter like this which doesn't need any extra parameters, you can also shorten it to:
```
await (
        await listFilesAndSubFoldersInFolder(buildMacroTestFolder)
      ).filter(isSasFile)
```

Reply:
wow, it's very nice!
thanks!